File: Data Structures & Algorithms/three-integer-sum/submission-3.java
class Solution {
    public List<List<Integer>> threeSum(int[] nums) {
        List<List<Integer>> ans = new ArrayList();
        Arrays.sort(nums);
        int sum;
        for (int i =0;i<nums.length-2;i++)
        {
            if(i>0 && (nums[i-1]==nums[i]))
            {
                continue;
            }
            int l= i+1;
            int r = nums.length-1;
            while(l<r)
            {
                sum = nums[i]+nums[l]+nums[r];
                if(sum==0)
                {
                    ans.add(Arrays.asList(nums[i],nums[l],nums[r]));
                    l++;
                    r--;
                    while(l>0 && nums[l]==nums[l-1] && l<r)
                    {
                        l++;
                    }
                }
                else if(sum<0)
                {
                    l++;
                }
                else
                {
                    r--;
                }

            }
        }
        return ans;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-3.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        longes = 0

        for i in nums:
            if i-1 in nums:
                continue
            else:
                c =0
                j =i
                while True:
                    if j in nums:
                        c = c+1
                        j = j+1
                    else:
                        break

                longes = max(c,longes)

        return longes